src/omnisharp/options.ts

```diff
@@ -207,53 +209,44 @@ export class Options {
         return excludePaths;
 
         function getExcludes(config: WorkspaceConfiguration, option: string): string[] {
-            let optionValue = config.get<{ [i: string]: boolean }>(option);
-            if (!optionValue) {
-                return [];
-            }
-
+            const optionValue = config.get<{ [i: string]: boolean }>(option, {});
             return Object.entries(optionValue)
                 .filter(([key, value]) => value)
                 .map(([key, value]) => key);
         }
     }
 
-    private static readPathOption(csharpConfig: WorkspaceConfiguration, omnisharpConfig: WorkspaceConfiguration): string | null {
+    private static readPathOption(csharpConfig: WorkspaceConfiguration, omnisharpConfig: WorkspaceConfiguration): string {
         if (omnisharpConfig.has('path')) {
             // If 'omnisharp.path' setting was found, use it.
-            return omnisharpConfig.get<string>('path');
+            return omnisharpConfig.get<string>('path', '');
         }
         else if (csharpConfig.has('omnisharp')) {
             // BACKCOMPAT: If 'csharp.omnisharp' setting was found, use it.
-            return csharpConfig.get<string>('omnisharp');
+            return csharpConfig.get<string>('omnisharp', '');
         }
         else {
-            // Otherwise, null.
-            return null;
+            // Otherwise, the empty string.
+            return '';
         }
     }
 
     private static readUseGlobalMonoOption(omnisharpConfig: WorkspaceConfiguration, csharpConfig: WorkspaceConfiguration): string {
-        function toUseGlobalMonoValue(value: boolean): string {
+        function toUseGlobalMonoValue(value?: boolean): string | undefined {
+            if (value === undefined) {
+                return undefined;
+            }
+
             // True means 'always' and false means 'auto'.
             return value ? "always" : "auto";
         }
 
-        if (omnisharpConfig.has('useGlobalMono')) {
```

Comment:
I'm not confident this fallback logic ever worked...I'm running the `singleCsproj` integration test suite right now which doesn't have `useGlobalMono` defined, but this still returns true (with `get` returning "auto", the default).

I suppose the upside is this whole section is going away soon, so I don't suppose it matters too much.

Reply:
This bit of code will be removed prior to the next release to address https://github.com/OmniSharp/omnisharp-vscode/issues/5120